myreactapp/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "myreactapp",
```

notre application est site-f38-studio
```suggestion
  "name": "myreactapp",
```